users/script.js

```diff
@@ -328,22 +329,24 @@ function clearFilters() {
   );
 }
 
-async function showUserDataList(
+const showUserDataList = async (
   page,
   userListElement,
   paginationElement,
   loaderElement,
   prevBtn,
   nextBtn,
-) {
+) => {
   try {
+    // Fetch users data with pagination parameters
     const userData = await getUsersData(page);
     if (userData.length) {
-      if (userData.length < 100) {
+      if (userData.length < USER_FETCH_COUNT) {
```

Comment:
where this defined USER_FETCH_COUNT?

Reply:
`USER_FETCH_COUNT` constant is defined in another file where constants are typically declared. It represents the maximum number of users to fetch per request.

Reply:
https://github.com/Real-Dev-Squad/website-dashboard/blob/develop/users/constants.js @vinit717 you can see this file for references.